packages/angular-compiler/src/lib/compile.ts

```
@@ -313,6 +331,37 @@ export function compile(
+              }
+              return out;
+            }
+            if (entry.kind === 'tuple' && entry.members) {
+              const out: string[] = [];
+              for (const name of entry.members) {
+                out.push(...collectExpandedImports(name, visited));
+              }
+              return out;
+            }
+            return [depClassName];
+          };
```

Biome (2.4.10)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)